feconf.py

```diff
@@ -25,7 +25,10 @@
 
 from constants import constants
 
-from typing import Dict, List, Text, Union # isort:skip # pylint: disable=unused-import
+from typing import Dict, List, NewType, Union # isort:skip # pylint: disable=unused-import
+
+TypeCommand = (
```

Comment:
Maybe just `Command` or `CommandType`.

Reply:
Changed it to `CommandType`. The reason here is that the variable name must show that a variable is a type. Thanks!